website/src/components/brand-picker/brand-picker.js

```diff
@@ -26,10 +38,14 @@ export const BrandPicker = () => {
 						<li
 							key={i}
 							css={{
-								flexBasis: '250px',
+								flexBasis: '100%',
+								'@media min-width(600px)': {
+									flexBasis: '100%',
+									maxWidth: 250,
```

Comment:
Let's make units explicit:
```suggestion
									maxWidth: '15.625rem',
```